frontend/src/app/AppFooter.tsx

```diff
@@ -3,7 +3,7 @@ import Footer, { FooterProps } from '~components/Footer'
 
 type AppFooterProps = Pick<
   FooterProps,
-  'variant' | 'containerProps' | 'compactMonochromeLogos'
+  'variant' | 'containerProps' | 'compactMonochromeLogos' | 'bgColorScheme'
```

Comment:
i know this is WIP but just reviewing first hehe. I don't think we need a new prop, since there already is `containerProps` that the parent can use